Networking15.02Escaping/Networking15.02Escaping/ImageService.swift

```diff
@@ -27,7 +28,7 @@ struct ImageService {
         ]
         let url = urlComponents.url
         let session = URLSession.shared
-        
+
         let dataTask = session.dataTask(with: url!) { data, response, error in
             if error == nil {
                 do {
```

Comment:
remove `DispatchQueue.main.async` from `Service` it's not a part of service on what queue do the job.